### DISCUSSION TITLE
redis

### DISCUSSION THREAD

**Discussion by KH93b** [nanoexpress/nanoexpress#255] [General] [answered]
Is there any possibility of scaling uWebSockets on multiple instances (for eg. using Redis)?

**Comment by dalisoft** [comment, marked as answer]
@KH93b I not sure what you mean, but i'm using this project with `pm2` and works fine on 16-core machine and it's very fast

**Reply by KH93b** [reply]
I mean horizontal scaling of servers.

**Reply by dalisoft** [reply]
Check out https://github.com/uNetworking/uTracker
